OWML.ModHelper.Menus/OwmlConfigMenu.cs

```diff
@@ -4,9 +4,8 @@ namespace OWML.ModHelper.Menus
 {
     public class OwmlConfigMenu : ModConfigMenuBase
     {
-        private const string GamePathTitle = "Game path";
         private const string VerboseModeTitle = "Verbose mode";
-        private const string BlockInputTitle = "Mod button combinations block game input";
+        private const string BlockInputTitle = "Mod input combinations block game input";
```

Comment:
This is still a little confusing... I haven't been keeping up with the combinations stuff, so I have no idea what this means

Reply:
Totally. I've had it explained several times and I still don't quite get it. I tried to make it more understandable here, but not sure if I succeeded.

Reply:
When it's on InputHandler would for example block `Shift` from game while registered combination `Shift+L` is being pressed

Reply:
Maybe we stop calling it "combinations"? It's trying to refer to the shortcuts that mods can define as a mod setting. As in, "if this setting is set to true, inputs used by mods will block the actions that the same shortcut would cause in the base game". For instance, if you had a shortcut that includes the jump button, pressing that shortcut would cause the player to jump, unless this option is enabled.

Some examples that might be clearer:
1. "Prevent game actions from being triggered by mod inputs"
2. "Mod inputs can block triggering of game actions"

Reply:
👍 I tried alt. 2